### ISSUE TITLE
Adding Netbox Plugins

### ISSUE THREAD

**Issue by sujayagit** [bootc/netbox-chart#122]
Is it possible to get more detailed document/examples about adding 'Netbox plugins' in netbox-chart.
In values.yaml it's mentioned:
# Enable installed plugins. Add the name of each plugin to the list.
plugins: []
How to install these plugins before adding here?

**Comment by rjhenry** [comment]
The easiest way to install them is to build a new image, as is noted here: https://github.com/bootc/netbox-chart/issues/118

**Comment by bootc** [comment]
Indeed, it's not feasible for the chart to influence the plugins available in the image. We deliberately run with a read-only root filesystem for security, for example, which makes it impossible to install plugins. You can probably override this if you really want to, but that's not something we recommend. The recommended route is to build your own customised image.

**Comment by sujayagit** [comment]
Ok..Thanks.
